### DISCUSSION TITLE
How to enter specific product information in order for this data to be sent via the API

### DISCUSSION THREAD

**Discussion by Alexdimirop** [AshleyRedman/WPClothes2Order#28] [Q&A] [answered]
Hi,

I am new to coding and don't have much php knowledge...

Through their docs I can see that in the JSON file example there is an array that contains information about the print position, width, a link for the image to be printed etc...

But I don't understand how the plugin collects all these information. I mean where should I put those information in order for the plugin to collect them and send the information through the api. Should I create new fields in product page?

**Comment by AshleyRedman** [comment, marked as answer]
Hi @Alexdimirop, this project is currently under development, so would highly advise on not trying to use in production. Version 0.1.0 which is the only released version currently works by placing WooCoomerce products into the auto created taxonomy. Though that process isn't as flexible as we would like, so the road map for this looks something like:

- On a WooCommerce product, you will have an option to 'mark product' as a C2O product. 
- Once marked, you will have options to enter the print type, position, width.
- The products main featured image URL will be used as the file link, but that can be overridden by a separate link. 

So back to your question, those options are yet to be added to this plugin, but that is the plan. Once a stable version 1.0.0 is released, tested & ready for production, we will be sending out a release notice for anyone who joined our mailing list here: https://www.wpclothes2order.com/ 
So if you haven't already, feel free to join the mailing list. In the meantime, feel free to look through the current main branch & if. you would like to contribute, this projects README should have all the information on how the get setup. 

Hope that helps!

**Comment by Alexdimirop** [comment]
Thank you for your response.

Ok now I understand how it works. It seems like an easy way to set up a product for C2O although I believe it will be a very difficult project to integrate C2O an non C2O products during checkout into a single order for the customer. 

Also do you think that it'll be able to retrieve all the information regarding C2O stock level of a product? That would be great.

I already joined the list and looking forward to hearing from you.

**Comment by AshleyRedman** [comment]
@Alexdimirop We do have some plans for the stock level, specifically we are looking at a daily cron job to pull down their stock levels & sync it with the local WooCommerce stock level. We have an open issue about it & I am currently tackling this feature. https://github.com/AshleyRedman/WPClothes2Order/issues/21
